_projects/image-tiff.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: project
+name: "tiff"
+title: "Tiff"
+weight: 700
+crates_io: "https://crates.io/crates/tiff"
+crates_io_badge: "https://img.shields.io/crates/v/tiff.svg"
+github: "https://github.com/image-rs/image-tiff"
+docs: "https://docs.rs/tiff"
+docs_badge: "https://docs.rs/tiff/badge.svg"
+title: "Tiff"
```

Comment:
Probably doesn't matter, but `title` appeared twice here. And also it should probably say "decoder and encoder" just bellow here as well.